qfieldsync/utils/file_utils.py

```diff
@@ -44,6 +44,10 @@ def fileparts(fn, extension_dot=True):
 
 
 def get_children_with_extension(parent, specified_ext, count=1):
+    if not os.path.isdir(parent):
+        raise QFieldSyncError(
+            "The directory {} could not be found".format(parent))
```

Comment:
tr?

Reply:
We could use QCoreApplication.translate() here (as well as the other QFieldSyncErrors in this file).

Reply:
Actually maybe not, since translate can't deal with arguments. Or is this going to be a TIL moment? :)